2016-06-20 14:25:20.884 -0600	START		Running Background.feature
2016-06-20 14:25:20.892 -0600	LogSuccess		I have <num> apples
2016-06-20 14:25:20.906 -0600	LogError		I give <num1> apples to a monkey
2016-06-20 14:25:20.906 -0600	Log		fail	7	on_i_am_left_with_var_apples	Background.script<SOURCE>
2016-06-20 14:25:20.908 -0600	LogSuccess		I am left with <num2> apples
2016-06-20 14:25:20.909 -0600	EndTestCase		(Duration:"0.023623", Errors:"2", Exceptions:"0", StartTime:"2016-06-20 14:25:20 -0600", Successes:"2", TestCase:"Background.feature", Warnings:"0")
2016-06-20 14:25:21.073 -0600	FAILURE	Screen_Error.png	1 scenarios (1 failed, 0 passed) 3 steps (1 failed, 0 skipped, 2 passed) Execution Time 0:00:00
